arch/risc-v/src/common/riscv_exception_common.S

```diff
@@ -69,48 +69,16 @@
 exception_common:
 
   addi       sp, sp, -XCPTCONTEXT_SIZE
```

Comment:
It was like this so I'll keep it like this

Reply:
Here is my thought: It isn't a problem before since we just have one exception entry pointer, but now we have two(exception_common and riscv_dispatch_syscall), it's always good to make both share the code base as much as possible.